app/src/main/java/com/sanjeevyadavit/magecart/presentation/MainActivity.kt

```diff
@@ -31,12 +32,13 @@ class MainActivity : AppCompatActivity() {
         binding = DataBindingUtil.setContentView(this, R.layout.activity_main)
         sharedPref = getSharedPreferences(
             SHARED_PREFERENCE_NAME,
-            Context.MODE_PRIVATE
+            MODE_PRIVATE
         )
         getNavController()
         setupBottomNavigation()
         setupToolbar()
         setupDrawer()
+        viewModel.storeConfigs // Hack to actually call the API
```

Comment:
What is being done here?

Reply:
I wanted to make an API call at app launch, and that data will be used at many fragments.

So I was facing issue, when I wasn't writing this line and trying to get the same instance of viewModel in my fragment, it was getting initialised there for first time and I was getting data null

Reply:
Can share the base url to a fragment via SharedPreference